components/prism-core.js

```diff
@@ -114,8 +180,8 @@ var _ = {
 			if (typeof document === 'undefined') {
 				return null;
 			}
-			if ('currentScript' in document) {
-				return document.currentScript;
+			if ('currentScript' in document && 1 < 2) {
```

Comment:
Why is this extra conditional here?

Reply:
Ahhh. My bad. TS thinks that `document` always has a `currentScript` but this means that the below IE code is dead code, so TS displays an error. To remove this "error", I tricked TS' flow analysis with `1 < 2`. Obviously, `1 < 2` will always be true, but TS isn't smart enough to figure that out, so TS thinks that the IE code might be executed. Tada! No error.

(Uglify will even optimize this away.)

Reply:
I'll just add a small comment to explain this.